compliance/unapproved_instance_types/unapproved_instance_types.pt

```diff
@@ -177,19 +199,16 @@ end
 ###############################################################################
 
 escalation "report_approved_instances" do
+  automatic true
+  label "Send Mail"
+  description "Sends incidents email"
   email $param_email
 end
 
-escalation "approve_stop_instances" do
-  request_approval  do
-  label "Approve Instance Stop"
-    description "Approve to stop instance with specified instance types."
-    parameter "approval_reason" do
-      type "string"
-      label "Reason for Approval"
-      description "Explain why you are approving the action."
-    end
-  end
+escalation "stop_instances" do
+  automatic false
+  label "Stop Instances"
+  description "Stop Instances"
```

Comment:
How about something like:  Stop selected instances with unapproved instance types.

Reply:
changed the description.